src/main.rs

```diff
@@ -6,23 +6,28 @@ mod session;
 mod utils;
 mod window;
 
+use crate::config::VERSION;
```

Comment:
This should go below the use declarations of the external crates (grouped separately from them).

To clarify, my preferred order and grouping for use declarations is:

1. Module imports
2. 'self' use declarations (which are useful to simplify the access to the inner objects of the modules)
3. Use declarations of external crates (including std)
4. Use declarations of the current crate

Reply:
I overlooked that one. Nevertheless, I removed the use statement altogether, since `VERSION` is only used once anyway. This looks cleaner then.